db/kv_dense_block.go

```diff
@@ -218,8 +218,8 @@ func (d *DenseBlock) addEntry(logEntry *LogEntry) (changed bool, removalRequired
 		// Ensure this entry hasn't already been written by another writer
 		clockSequence := d.getClock()[logEntry.VbNo]
 		if logEntry.Sequence <= clockSequence {
-			base.LogTo("ChannelStorage+", "Index already has entries later than or matching sequence - skipping.  key:[%s] seq:[%d] index_seq[%d] blockKey:[%s]",
-				logEntry.DocID, logEntry.Sequence, clockSequence, d.Key)
+			base.LogToR("ChannelStorage+", "Index already has entries later than or matching sequence - skipping.  key:[%s] seq:[%d] index_seq[%d] blockKey:[%s]",
+				base.UD(logEntry.DocID), logEntry.Sequence, clockSequence, base.UD(d.Key))
```

Comment:
`d.Key` sometimes not redacted (e.g. line 161, 571, 582), but it looks like it's a SGW-generated value. Maybe no need for `base.UD` here?

Reply:
Yeah, from what I could tell, the key here is just a pointer to the block document, and the key itself won't have any possibility of user data.  Removed this `base.UD()` call.